lunavl/sdk/luna_faces.py

```diff
@@ -1,4 +1,5 @@
-"""Module realize hight level api for estimate face attributes
+"""
+Module realize hight level api for estimate face attributes
```

Comment:
`High-level api for estimating face attributes`

Reply:
++